addons/spreadsheet/static/src/chart/odoo_chart/odoo_bar_chart.js

```diff
@@ -62,7 +62,7 @@ function getBarConfiguration(chart, labels, locale) {
     const legend = {
         ...config.options.legend,
         display: chart.legendPosition !== "none",
-        labels: { fontColor },
+        labels: { color: fontColor },
```

Comment:
you can even go a step further and rename the variable to  `color` as well, then you could write 
```suggestion
        labels: { color },
```